### ISSUE TITLE
https://dicekeys.app/ unusable on 2019 MacBook Pro w/Chrome 85.0.4183.102

### ISSUE THREAD

**Issue by yeliaBdE** [dicekeys/beta-program#15]
When the "Scan Your DiceKey" button is pressed, an extremely dark image is displayed; the image is so dark that only the very brightest highlights are visible as dark gray, with everything else in the image being black.

Furthermore, the image is static; it never changes, even if the page is left open for 10-15 minutes. To test this, I covered the camera lens and the image remained unchanged. I also tried it the other way around (by pressing the "scan" button with the camera lens covered,) and the resulting image was pure black, and never updated after the lens was uncovered.

This was using the MacBook's internal camera, by the way.

I also have a camera in an attached display (an old Apple Thunderbolt Display). When trying to use that camera (by changing the dropdown to select it, the camera goes active (i.e., its green light illuminates), but the image remains unchanged from that captured by the MacBook's internal camera.

tl;dr: Successful scanning of Dicekeys impossible due to: 1. Severe underexposure resulting in an almost entirely black image, and 2. No updates to the image after the one captured initially makes proper alignment of DiceKeys impossible.

**Comment by yeliaBdE** [comment]
Using the staging version of the app I was able, after a fair bit of struggle, to get my DiceKey scanned using my MacBook's internal camera. The image recognition was extremely sensitive to movement--I'd see an array of green and red markers on the dice, and only by holding the DiceKey completely still with two hands resting solidly on the desk would a scan complete; often with it needing me to confirm a die that it just couldn't identify.

I am still unable to use the display's camera to successfully scan; a video stream with a decent frame rate is displayed, but the overall image seems to still be slightly unfocused, and so it's impossible to get any of the dice identified.

**Comment by UppaJung** [comment]
Are you able to submit a scree shot of the display camera use case so I can see all the OS and resolution details?

**Comment by yeliaBdE** [comment]
@UppaJung  Sure thing:

![Screen Shot 2020-09-25 at 12 55 22](https://user-images.githubusercontent.com/17483474/94295209-e4c64900-ff2e-11ea-9c3b-3437bf68ef3e.png)
